Gemfile

```diff
@@ -1,5 +1,9 @@
 source 'https://rubygems.org'
 
+%w[rspec rspec-core rspec-expectations rspec-mocks rspec-support].each do |lib|
+  gem lib, :git => "git://github.com/rspec/#{lib}.git", :branch => 'master'
+end
```

Comment:
Hi @packrat386 ! Thanks for your review, you are right! And as I see you already have a PR & commit about it #119 😄 👌